_database-integrations/mysql/azure/microsoft-azure-mysql-v1.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Microsoft Azure MySQL (v1)
+keywords: microsoft azure, azure, database integration, etl azure, azure etl
+permalink: /integrations/databases/microsoft-azure-mysql/v1
+summary: "Connect and replicate data from your Microsoft Azure SQL database using Stitch's Microsoft Azure MySQL integration."
+show-in-menus: true
+
+key: "microsoft-azure-mysql-integration"
+
+# -------------------------- #
+#     Integration Details    #
+# -------------------------- #
+
+name: "microsoft-azure-mysql"
+display_name: "Microsoft Azure MySQL"
+
+hosting-type: "microsoft-azure"
+
+this-version: "1"
+
+driver: |
+  [PyMySQL 0.9.3](https://pymysql.readthedocs.io/en/latest/){:target="new"}
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+singer: true
+repo-url: https://github.com/singer-io/tap-mysql
+certified: true
+
+tap-name: "microsoft-azure-mysql"
```

Comment:
Remember - this is the *display name* of the tap backing the integration. In this case, it's `MySQL`.